openedx/core/lib/completion.py

```diff
@@ -0,0 +1,25 @@
+"""Helper methods for completion."""
+
+from xblock.completable import XBlockCompletionMode
```

Comment:
Should these methods be part of the https://github.com/edx/completion repo?

Reply:
@jcdyer Should these methods be moved onto the `CompletionService`? I see there are a couple of similar helper methods on it.

Reply:
@symbolist The CompletionService looks like it would be a good place for it.  :+1:

Reply:
Moved to `CompletionService`.